e2e/config/playwright.config.common.ts

```diff
@@ -66,12 +66,9 @@ export function config(config: CustomEnvConfig) {
       grafanaAPICredentials: getGrafanaUser(),
       /* Base URL to use in actions like `await page.goto('/')`. */
       baseURL: getGrafanaUrl(),
-      // Record trace only when retrying a test for the first time.
       screenshot: 'only-on-failure',
-      // Record video only when retrying a test for the first time.
-      video: 'on-first-retry',
-      /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-      trace: 'on-first-retry',
+      video: 'retain-on-failure',
+      trace: 'retain-on-failure',
```

Comment:
see comment for the test about this

Reply:
That's interesting! Do we do any other e2e tests for failed queries or errors that are returned from Prometheus in a panel?